client/main/variableeditor.js

```diff
@@ -221,15 +221,16 @@ const EditorWidget = Backbone.View.extend({
         this.$el.empty();
         this.$el.addClass('silky-variable-editor-widget');
 
-        this.$title = $('<input class="silky-variable-editor-widget-title" type="text">').appendTo(this.$el);
+        this.$title = $('<input class="silky-variable-editor-widget-title" type="text" maxlength="50">').appendTo(this.$el);
         this._currentKeyboardContext = '';
         this.$title.focus(() => {
             this._currentKeyboardContext = keyboardJS.getContext();
             keyboardJS.setContext('');
             this.$title.select();
         } );
         this.$title.on("change keyup paste", () => {
-            this.model.set({ name: this.$title.val() });
+            let newName = this.$title.val().trim();
+            this.model.set({ name: newName });
         } );
```

Comment:
clicking the tick applies the changes but doesn't close the window. The up arrow closes the window. So i can apply the changes but it doesn't close the window.